lib/client/jsx/components/manifest/consignment_result.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import VectorResult from './vector_result';
+import MatrixResult from './matrix_result';
+import Vector from '../../models/vector';
+import Matrix from '../../models/matrix';
+
+const isPrimitiveType = (value)=> (
+  typeof value === 'string' || typeof value === 'number' ||
+  typeof value === 'boolean' || typeof value === 'undefined' ||
+  typeof value === null
+);
+
+const isVector = (value) => (
+  value instanceof Vector && isPrimitiveType(value(0))
+);
+
+const isMatrix = (value) => (value instanceof Matrix);
+
```

Comment:
I don't think we support nested things in vectors, anymore.

Reply:
Hmm, this appears to have been lost accidentally in c7e16c7dfaf3aef2e0a3377ad99924ee889839b1. I will attempt to put it back using code from before this commit.